cmd/cli/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	_ "embed"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"pantheon.io/edrt-policy-docs-functions/config"
+	firestore "pantheon.io/edrt-policy-docs-functions/internal/store"
+)
+
+var firestoreProject string
+
+func init() {
+	configMap := config.GetConfig()
+	viper.MergeConfigMap(configMap)
+	firestoreProject = viper.GetString("FIRESTORE_PROJECT")
+}
+
+func main() {
```

Comment:
I do not know what plans we have for this cli. For now, it looks like we can check documents straight in the firestore console by using the web ui.